### PR TITLE
Sfl 263

### PR DESCRIPTION
**Pull request by vineet-webonise**

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/PlannerVC/Planner+TableView.swift [hunk 1]
```diff
@@ -84,8 +84,18 @@ extension PlannerVC: UITableViewDelegate, UITableViewDataSource {
     }
     
     @objc func startVideo(sender: UIButton) {
-        editCloseBtnTapped(sender: sender)
-        playVideo(row: sender.tag)
+        editCloseBtnTapped(sender: sender)        
+        switch sender.titleLabel?.text {
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/PlannerVC/Planner+TableView.swift:88]: Create Separate Method And call that method in switch case 
just like ` playVideo(row: sender.tag)`
for 
```
dictEditData[Constants.EditData.NEW_CONTENT_ID] = plannerSubsection[sender.tag].id
             editActionVM.dictEditActionValues = dictEditData
             editActionVM.swapContentApiCall(view: self.view)
```

File: SUNFLOWR/SUNFLOWR/PlannerVC/PlannerVC.swift [hunk 2]
```diff
@@ -165,8 +165,20 @@ extension PlannerVC: AddSwapDelegate {
...
     /// Swap/Add Btn clicked on popover
     /// - Parameter selectedIndex: selected Index
-    func didTapOnDoneBtn(selectedIndex: [Int]) {
+    func didTapOnDoneBtn(selectedIndex: [Int], selectedActivity: String) {
         hidePopOver()
+        if let newSubSectionId = addSwapPopOver.selectedSubSectionId{
+            dictEditData[Constants.EditData.NEW_SECTION_ID] = newSubSectionId
+        }
+        switch selectedActivity {
+        case Constants.ActivityName.Swap :
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/PlannerVC/PlannerVC.swift:174]: same here

File: SUNFLOWR/SUNFLOWR/PlannerVC/Planner+TableView.swift (hunk 1 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/PlannerVC/Planner+TableView.swift:88]: Done

File: SUNFLOWR/SUNFLOWR/PlannerVC/PlannerVC.swift (hunk 2 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/PlannerVC/PlannerVC.swift:174]: Done

**Pull request merged**